mycroft/tts/tts.py

```diff
@@ -2,10 +2,13 @@
 from ovos_plugin_manager.templates.tts import PlaybackThread, \
     TTS, TTSValidator, EMPTY_PLAYBACK_QUEUE_TUPLE
 from mycroft.configuration import Configuration
+from mycroft.tts.dummy_tts import DummyTTS
 
 
 class TTSFactory(OVOSTTSFactory):
     @staticmethod
     def create():
         config = Configuration.get()
-        return OVOSTTSFactory.create(config)
\ No newline at end of file
+        if config.get("tts", {}).get("module", "") == "dummy":
+            return DummyTTS()
```

Comment:
this was missed in #1 

a dummy plugin should be made in the future and native support added to OPM, after that `dummy_tts.py` can be deleted